atomics/t1118/t1118.yaml

```diff
@@ -0,0 +1,19 @@
+attack_technique: T1118
+display_name: InstallUtil
+
+atomic_tests:
+- name: InstallUtil uninstall method call
+  description: |
+    Executes the Uninstall Method
+  supported_platforms:
+    - windows
+  input_arguments:
+    filename:
+      description: location of the payload
+      type: Path
+      default: T1118.dll
+  executor:
+    name: command_prompt
+    command: |
+      C:\Windows\Microsoft.NET\Framework\v4.0.30319\csc.exe /target:library T1118.cs
```

Comment:
@caseysmithrc are these hardcoded to 4.0.30319 or should it be any version that exists on the system?

Reply:
For this we could make that a variable.